admin/acceptance.py

```diff
@@ -631,6 +629,16 @@ def _runner_VAGRANT(self, package_source,
             configuration, this is ignored.
         :returns: ``VagrantRunner``
         """
+        # By default Vagrant runner just uses ZFS, which is pre-configured
+        # in the Vagrant images, so additional configuration may not be
+        # available:
+        try:
+            dataset_backend = self.dataset_backend()
+            dataset_backend_config = self.dataset_backend_configuration()
+        except KeyError:
+            dataset_backend = DatasetBackend.zfs
+            dataset_backend_config = {}
```

Comment:
I'm hoping that `"backend": "zfs"` is required here and that accounts for some part of the acceptance test failures.